src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC } from "react";
+import { Loader } from "@mantine/core";
+import { useGetTodosQuery } from "../../redux/apiSlice";
+import { Todo } from "../Todo/Todo";
+import { StyledTodoWrapper } from "./TodoListStyles";
+
+export const TodoList: FC = () => {
+  const {
+    data = [],
+    isLoading,
+    isSuccess,
+    isError,
+    isFetching,
+    error,
+  } = useGetTodosQuery();
+  let content = null;
+
+  if (isLoading) {
```

Comment:
Zkoušel jsem si to pustit a ten loading se ti zobrazuje jen na první načtení. Udělal bych to tak, aby se zobrazil při každém načtení.